### DISCUSSION TITLE
Работа приложения с вебкам-сайтами и торрентами

### DISCUSSION THREAD

**Discussion by Olezha180** [anticensority/runet-censorship-bypass#312] [Q&A]
Утром торрент-сайты и вебкам-сайты открывались, сейчас - нет, чё за фигня? Что происходит?

**Comment by ilyaigpetrov** [comment]
РКН блочит нас каждый день. Каждый день мы меняем IP-адреса прокси-серверов.
Instagram, Twitter, Facebook временно не проксируются.
Заглядывайте в новости:
1) Про ежедневные блокировки: https://github.com/anticensority/runet-censorship-bypass/issues/10#issuecomment-2356186450.
2) Про Инсту, Твиттер, Фэйсбук: https://github.com/anticensority/runet-censorship-bypass/issues/10#issuecomment-2364286599.

Ответная мера на ежедневную блокировку РКН зависит от разработчика проекта "АнтиЗапрет".
Он немного занят, но я ему напоминаю время от времени об этой задаче.
